Toma/HW3/R_OSPool.submit

```diff
@@ -0,0 +1,27 @@
+# File Name: R.submit
++SingularityImage = "/cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo-r:3.5.0"
+
+# Specify your executable and arguments
+executable = spill_calculation.r
+arguments = $(csvfile)
+
+# Tell HTCondor what to name log, standard error, and standard output("screen output") files
+log = log/spill.$(Cluster).$(Process).log
+output = errout/spill.$(Cluster).$(Process).out
+error = errout/spill.$(Cluster).$(Process).error
+
+#Specify the job duration
++JobDurationCategory = "Medium"
+
+# Specify that HTCondor should transfer files to and from the
+#  computer where each job runs. 
+transfer_input_files = 
```

Comment:
@maksudatoma you are not transferring any files. This submit file won't work. Moreover, HW3 asks you to copy the results(the results that you will get when your job runs successfully on OSPool) as well which you haven't included in this **PR**. Have you tried submitting the job (*condor_submit R_OSPool.submit*)?

Reply:
I have changed the submitted file with a new branch Toma-patch03. Let me know if further changes are needed.